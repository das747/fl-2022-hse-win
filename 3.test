=> &S& -> &\& A \&& &  B \\& @ test with \@ and \& @
&\& A \&& -> @Yet @
&  B \\& -> @another @
