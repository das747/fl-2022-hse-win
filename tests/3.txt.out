LexToken(START,'=>',11,0)
LexToken(NON_TERM,'S',11,3)
LexToken(RULE,'->',11,7)
LexToken(NON_TERM,'& A &',11,10)
LexToken(NON_TERM,'  B \\',11,20)
LexToken(TERM,' test with @ and & ',11,29)
LexToken(NON_TERM,'& A &',12,53)
LexToken(RULE,'->',12,63)
LexToken(TERM,'Yet ',12,66)
LexToken(NON_TERM,'  B \\',13,73)
LexToken(RULE,'->',13,82)
LexToken(TERM,'another ',13,85)

Start non-terminal:
	'S'

Non-terminals:
	'  B \'
	'S'
	'& A &'

Terminals:
	'another '
	' test with @ and & '
	'Yet '

Rules:
	'S' -> '& A &' '  B \' ' test with @ and & '
	'& A &' -> 'Yet '
	'  B \' -> 'another '

Grammar is not in Chomsky normal form
